install/kubernetes/cilium/values.yaml

```diff
@@ -1286,6 +1286,9 @@ hubble:
         extraDnsNames: []
         # -- extra IP addresses added to certificate when its auto gen
         extraIpAddresses: []
+        # DNS name used by the backend to connect to the relay
+        # For GKE Dataplane V2 this should be set to relay.kube-system.svc.cluster.local
+        relayName: "ui.hubble-relay.cilium.io"
```

Comment:
Elsewhere, we called this `clusterDomain`, e.g. here:
https://github.com/cilium/cilium/blob/fd490f6bdef23ef0501687237b6ea721c5ff82e9/install/kubernetes/cilium/values.yaml.tmpl#L1056

It is then used together with the release namespace to build the service name as follows:
https://github.com/cilium/cilium/blob/692fe68d08090d6f8e9a890c3fcd9db475e31012/install/kubernetes/cilium/templates/hubble-relay/configmap.yaml#L15

I think it would make sense to keep the two options as similar as possible

Reply:
@gandro - I have run the `make` commands provided and aligned the key name as suggested which I think is what you meant in the comment above but if not please let me know and I can amend.